pkg/kubelet/secret/secret_manager.go

```diff
@@ -201,8 +201,12 @@ type cachingSecretManager struct {
 }
 
 func NewCachingSecretManager(kubeClient clientset.Interface) (Manager, error) {
+	// TODO: Currently apiserver completely cannot keep up with 1 minute ttl in
+	// 5000-node clusters. So before we optimize apiserver and/or etcd, increase
+	// the TTL to 5 minutes (which is a lot, but...)
+	ttl := 5 * time.Minute
```

Comment:
Hmm - after introducing this, the test didn't blown up completely at the moment when it previously blown.
That said, I think that having the "SecretManager" abstraction is useful anyway, even if we decide to change the implementation completely in few weeks.

But going back - does increasing the ttl to e.g. 2m30s or sth like that sounds reasonable to you?